rust-toolchain

```diff
@@ -0,0 +1 @@
+1.75
```

Comment:
We should probably set MSRV in the workspace `Cargo.toml` file and I'm curious if there is a way to ensure that one and this one are in sync.

Reply:
Will add it to the cargo.toml, i can check but  I do not think that there is a standard way to check that both versions match.

Reply:
After doing a quick research i have not found a standard way of checking that all crates match the version of the rust-toolchain file. Hence decided to create a PR to address that and fix your comments:
https://github.com/0xPolygonMiden/miden-node/pull/272